src/test/java/org/javawebstack/orm/test/shared/util/QueryStringUtil.java

```diff
@@ -0,0 +1,89 @@
+package org.javawebstack.orm.test.shared.util;
+
+import java.util.Locale;
+
+import static org.javawebstack.orm.test.shared.knowledge.QueryKnowledgeBase.quoteCharacters;
+import static org.javawebstack.orm.test.shared.knowledge.QueryKnowledgeBase.sectionNames;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class QueryStringUtil {
+
+    String queryString;
+
+    public QueryStringUtil(String queryString) {
+        this.queryString = queryString;
+    }
+
+    public String getSectionByName(String inputSectionName) {
```

Comment:
keywords like `JOIN` can recur, so this method should maybe return a list or take an index?

Reply:
Totally forget about that possibility. I think a list makes the most sense. Will rework that part and adjust and add tests relating to that.

Reply:
Has been changed